### PR TITLE
Release 3.8 staging

### PR DESCRIPTION
**Pull request by ramijebara**
Various updates to the 3.8 release docs.

### CODE REVIEW

**Review by Thomathan — APPROVED** [review]

File: documentation/settings/topics/service-now-cmdb-sync.rst [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+     - String
+     - No
+   * - power_providing_asset_ids
+     - u_power_providing_asset_ids
+     - String
+     - No
+
+Note that when there are ID and name fields returned as an array, the sorting order will be the same. For example, if the asset has two power-providing assets, the name and ID order in their respective arrays will be the same. This is done to make it easier to write data reconciliation rules on the ServiceNow side.
+
+ImportSet JSON Example
```
Comment (kyledickau) [inline documentation/settings/topics/service-now-cmdb-sync.rst:150]: Should be a lower case "e" in "Example".

File: documentation/settings/topics/integrations.rst [hunk 2]
```diff
@@ -5,214 +5,13 @@
 Integrations
 ************
 
-Hyperview Integrations are a separately licensed set of features that let you push or pull data from external systems such as ServiceNow. Each integration is separately licensed.
+Hyperview APIs are documented from within the product and on the Hyperview Documentation site. Customers are free to write their integrations and there are API integration Samples provided on `GitHub <https://github.com/HyperviewHQ/API-Samples>`_.
```
Comment (kyledickau) [inline documentation/settings/topics/integrations.rst:8]: Should be a lowercase "s" in, "API integration Samples".

**Review by kyledickau — APPROVED** [review]
Just a few small formatting changes needed but I approve.

File: documentation/settings/topics/service-now-cmdb-sync.rst (hunk 1 above)
Comment (ramijebara) [reply documentation/settings/topics/service-now-cmdb-sync.rst]: Fixed, Thanks!

File: documentation/settings/topics/integrations.rst (hunk 2 above)
Comment (ramijebara) [reply documentation/settings/topics/integrations.rst]: Fixed. Thanks!

**Pull request merged**